packages/manager/modules/pci/src/projects/project/storages/cold-archive/containers/manage-archive/manage-archive.routing.js

```diff
@@ -0,0 +1,36 @@
+import {
+  COLD_ARCHIVE_TRACKING_PREFIX,
+  COLD_ARCHIVE_STATES,
+  REGION,
+} from '../../cold-archives.constants';
+
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state(COLD_ARCHIVE_STATES.CONTAINERS_CONTAINER_MANAGE, {
+    url: '/manage',
+    views: {
+      modal: {
+        component: 'pciStoragesColdArchiveContainersManageContainer',
+      },
+    },
+    layout: 'modal',
+    resolve: {
+      breadcrumb: () => null,
+      endpoint: /* @ngInject */ (PciStoragesColdArchiveService, projectId) => {
+        try {
+          return PciStoragesColdArchiveService.getArchiveEndpoint(
+            projectId,
+            REGION,
+          ).then(({ data }) => {
+            return data?.services[0]?.endpoint;
+          });
+        } catch (err) {
+          return '';
+        }
+      },
```

Comment:
As your function is not marked with `async` please use the promise version of `$http.get`
Also, the `return` statements can be omitted for readability 
```suggestion
      endpoint: /* @ngInject */ (PciStoragesColdArchiveService, projectId) =>
        PciStoragesColdArchiveService.getArchiveEndpoint(projectId, REGION)
          .then(({ data }) => data?.services[0]?.endpoint)
          .catch(() => ''),
```

Reply:
Done.